src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -1,11 +1,33 @@
 package bio.terra.cli.command.app.passthrough;
 
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.PassthroughException;
+import bio.terra.cli.exception.UserActionableException;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra gcloud" command. */
 @Command(name = "gcloud", description = "Call gcloud in the Terra workspace.")
 public class Gcloud extends ToolCommand {
 
+  private static final Logger logger = LoggerFactory.getLogger(Git.class);
+
+  @CommandLine.Option(
```

Comment:
can we hide this option if cloudbuild is not enabled in the env? 
https://picocli.info/#_model_transformations